site/content/integrate/apps/call/_index.md

```diff
@@ -79,7 +79,6 @@ Depending on the location and expansions, calls will have different context. The
 | `channel_id`               | string   | ID from the channel from within which the call was performed.                       |
 | `post_id`                  | string   | ID from the post from within which the call was performed.                          |
 | `root_post_id`             | string   | If the call was performed from a post in a thread, the root post ID of that thread. |
-| `mattermost_site_url`      | string   | Mattermost base URL.                                                                |
```

Comment:
Why was this removed?

Reply:
It is duplicated in the table.

https://github.com/mattermost/mattermost-developer-documentation/blob/master/site/content/integrate/apps/call/_index.md?plain=1#L82-L84

Reply:
Missed that